src/sign/participant.rs

```diff
@@ -76,20 +83,15 @@ impl Input {
     }
 }
 
-/// ECDSA signature on a message.
-///
-/// When generated by this library, the signature will be produced by the
-/// threshold ECDSA algorithm by Canetti et al.
-#[derive(Debug)]
-pub struct Signature(k256::ecdsa::Signature);
-
 /// Protocol status for [`SignParticipant`].
 #[allow(unused)]
 #[derive(Debug, PartialEq)]
 pub enum Status {
-    /// Participant has been initialized.
+    /// Participant is created but has not received a ready message from self.
+    NotReady,
+    /// Participant received a ready message and is executing the protocol.
     Initialized,
-    /// Participant has finished the sub-protocol.
+    /// Participant finished the protocol.
     TerminatedSuccessfully,
 }
```

Comment:
note: I'm handling status a little differently in this module than in other modules. When we refactored how we handle ready messages, we added a separate `bool` type to all the other participants to track whether they're ready to do stuff yet. While looking at that here, I realized it's weird to have two different things tracking statuses that are related to each other. This isn't a fully-fledged state machine but at least it now captures the three basic statuses, and we can check them as we update.